UnityProject/Assets/Scripts/UI/Systems/AdminTools/GUI_AdminTools.cs

```diff
@@ -60,28 +67,10 @@ public void ToggleOnOff()
 			gameObject.SetActive(!gameObject.activeInHierarchy);
```

Comment:
I'd assume a method called ``ClosePanel`` will always set the active state to false rather than invert the current state.

Reply:
you are confusing `ToggleOnOff()` with `ClosePanel()`